server/paymentProviders/transferwise/index.ts

```diff
@@ -59,10 +61,13 @@ async function getTemporaryQuote(
   payoutMethod: PayoutMethod,
   expense: Expense,
 ): Promise<QuoteV2> {
+  expense.collective = expense.collective || (await models.Collective.findByPk(expense.CollectiveId));
+  expense.host = expense.host || (await expense.collective.getHostCollective());
+  const rate = await getFxRate(expense.currency, expense.host.currency);
   return await transferwise.getTemporaryQuote(connectedAccount, {
-    sourceCurrency: expense.currency,
+    sourceCurrency: expense.host.currency,
     targetCurrency: <string>payoutMethod.unfilteredData.currency,
-    sourceAmount: expense.amount / 100,
+    sourceAmount: (expense.amount * rate) / 100,
```

Comment:
To make what we're doing here more explicit and prevent any floating-point error.

```suggestion
    sourceAmount: centsAmountToFloat(expense.amount * rate),
```

An example of what we're trying to avoid here:
> \> (3*100.1)/100
> 3.0029999999999997